src/error.rs

```diff
@@ -35,16 +35,12 @@ pub struct Error {
 }
 
 impl Error {
-    pub(crate) fn new(e: Arc<ErrorKind>) -> Error {
-        Error {
-            kind: e,
-            labels: Vec::new(),
-        }
-    }
-
     pub(crate) fn pool_cleared_error(address: &StreamAddress) -> Self {
         ErrorKind::ConnectionPoolClearedError {
-            message: format!("Conneciton pool for {} cleared during operation execution", address)
+            message: format!(
```

Comment:
There are a number of format changes that seems to have been made in files where other changes were made. I'm not sure why the format checking script didn't report anything when running rustfmt again seems to change the code.

Reply:
Tiny nit, but while we're here, can you fix the typo in this message (connection spelled incorrectly)?